.github/workflows/new_version.yml

```diff
@@ -1,24 +0,0 @@
----
```

Comment:
This workflow was deleted without a substitute?

Reply:
This worklfow was from the original repository. I think we can a a simular workflow after reference our repository in the original one.